projects/storefrontapp-cds-integration/src/app/app.server.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
```

Comment:
since we have this support for ssr, we should add a script in package.json, similar to this: https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/develop/package.json#L52